.ci/run_docker.sh

```diff
@@ -1,13 +0,0 @@
-#!/usr/bin/env bash
-
-.ci/before_install_linux.sh
-
-if [[ $TRAVIS_BRANCH == "master" ]]; then
-  DOCKER_BUILDKIT=1 docker build --progress plain -t $DOCKERHUB_REPO:$COMMIT -t $DOCKERHUB_REPO:latest . 
```

Comment:
What was happening here is that we were tagging ***master builds only*** with `latest` on docker hub. I think this is nice to have and should be kept somehow, so that we do not end up with thousands of builds with commits and we don't actually know which one is the latest from `master` - or is this solved differently now?

Reply:
Just added a line in after success script to take care of this part